pages/02.documentation/06.node-deployments/default.en.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Node Deployments
+published: true
+taxonomy:
+    tag:
+        - nodes
+        - kubernetes
+---
+
+## Node Deployments
+
+MetaKube Clusters consist of master components and worker nodes. The worker nodes are managed through node deployments, which can be added, updated and removed through the API and UI.
+
+When creating a new cluster one node deployment will be added automatically. Nodes within one node deployment are each configured similar.
+
+[This tutorial](../../04.tutorials/09.manage-node-deployments/default.en.md) describes how you can edit the node deployments.
+
+## Settings
```

Comment:
Please add a screenshot for this. It is unclear how these settings are being set.

Reply:
The linked tutorial already covers how you can access (and where to edit) these settings